views/js/review/provider/tpl/qtiTestReviewProvider.tpl

```diff
@@ -1,4 +1,6 @@
 <main class="test-review-scope">
+    <a class="jumplink navigationlink" href="#" tabindex="1">{{__ "Jump to:"}} <b>{{__ "Navigation"}}</b></a>
+    <a class="jumplink answerlink" href="#" tabindex="1">{{__ "Jump to:"}} <b>{{__ "Question"}}</b></a>
```

Comment:
I don't get the point for those jumplinks. Why not use anchors instead?

Reply:
Because of the need to apply a style to an area when jumplink is in focus
And after click the need to set focus on the first navigatable element